mbuild/compound.py

```diff
@@ -523,6 +524,120 @@ def update_coordinates(self, filename):
+               Vandermeersch, T.; Hutchison, G.R. "Open Babel: An open
+               chemical toolbox." (2011) J. Cheminf. 3, 33
+        .. [2] Open Babel, version X.X.X http://openbabel.org, (installed
+               Month Year)
+
+        If using the 'MMFF94' force field please also cite the following:
+        .. [3] T.A. Halgren, "Merck molecular force field. I. Basis, form,
+               scope, parameterization, and performance of MMFF94." (1996)
+               J. Comput. Chem. 17, 490-519
+        .. [4] T.A. Halgren, "Merck molecular force field. II. MMFF94 van der
+               Waals and electrostatic parameters for intermolecular
+               interactions." (1996) J. Comput. Chem. 17, 520-552
+        .. [5] T.A. Halgren, "Merck molecular force field. III. Molecular
+               geometries and vibrational frequencies for MMFF94." (1996)
+               J. Comput. Chem. 17, 553-586
+        .. [6] T.A. Halgren and R.B. Nachbar, "Merck molecular force field.
+               IV. Conformational energies and geometries for MMFF94." (1996)
+               J. Comput. Chem. 17, 587-615
+        .. [7] T.A. Halgren, "Merck molecular force field. V. Extension of
+               MMFF94 using experimental data, additional computational data,
+               and empirical rules." (1996) J. Comput. Chem. 17, 616-641
+
+        If using the 'UFF' force field please also cite the following:
+        .. [3] Rappe, A.K., Casewit, C.J., Colwell, K.S., Goddard, W.A. III,
+               Skiff, W.M. "UFF, a full periodic table force field for
+               molecular mechanics and molecular dynamics simulations." (1992)
+               J. Am. Chem. Soc. 114, 10024-10039
+
+        If using the 'GAFF' force field please also cite the following:
+        .. [3] Wang, J., Wolf, R.M., Caldwell, J.W., Kollman, P.A., Case, D.A.
+               "Development and testing of a general AMBER force field" (2004)
+               J. Comput. Chem. 25 1157-1174
+        """
+        try:
+            import openbabel
+        except ImportError:
+            warn("Open Babel package not detected. Energy minimization will not "
+                 "be performed.", ImportWarning)
+            return
+
+        tmp_dir = tempfile.mkdtemp()
+        original = clone(self)
+        self.tweak()
+        self.save(os.path.join(tmp_dir,'un-minimized.pdb'))
+        obConversion = openbabel.OBConversion()
+        obConversion.SetInAndOutFormats("pdb", "mol2")
+        mol = openbabel.OBMol()
+
+        obConversion.ReadFile(mol, os.path.join(tmp_dir, "un-minimized.pdb"))
+
+        if forcefield == 'MMFF94':
+            ff = openbabel.OBForceField.FindForceField("MMFF94")
+        elif forcefield == 'UFF':
+            ff = openbabel.OBForceField.FindForceField("UFF")
+        elif forcefield == 'GAFF':
+            ff = openbabel.OBForceField.FindForceField("GAFF")
+        else:
+            forcefield = 'UFF'
```

Comment:
This shouldnt require so many cases. Something like:
```python
try:
   ff = openbabel.OBForceField.FindForceField(forcefield)
except ErrorRaisedByOpenBabelWhenThisFails:
    raise  # I think you should just raise the error here. Right now the code keeps going even if it can't do anything.
```